crypto/fipsmodule/rsa/rsa.c

```diff
@@ -916,3 +919,16 @@ int RSA_flags(const RSA *rsa) { return rsa->flags; }
 int RSA_blinding_on(RSA *rsa, BN_CTX *ctx) {
   return 1;
 }
+
+int RSA_pkey_ctx_ctrl(EVP_PKEY_CTX *ctx, int optype, int cmd, int p1, void *p2)
```

Comment:
Why did you choose you put this function in `rsa.c`?

Reply:
I put in `rsa.h` and then implemented it in the corresponding c file. I think this method is rsa specific.

Reply:
I renamed the method to `EVP_RSA_PKEY_CTX_ctrl` and moved to `crypto/evp/internal.h` and `p_rsa.c`.